tests/os/linux/snmp/load.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Documentation       Check load table
+
+Resource            ${CURDIR}${/}..${/}..${/}..${/}resources/import.resource
+
+Test Timeout        120s
+
+
+*** Variables ***
+${CMD}      ${CENTREON_PLUGINS} --plugin=os::linux::snmp::plugin
+
+
+*** Test Cases ***
+load ${tc}
+    [Tags]    os    linux
+    ${command}    Catenate
+    ...    ${CMD}
+    ...    --mode=load
+    ...    --hostname=${HOSTNAME}
+    ...    --snmp-version=${SNMPVERSION}
+    ...    --snmp-port=${SNMPPORT}
+    ...    --snmp-community=os/linux/snmp/linux
+    ...    --snmp-timeout=1
+    ...    ${extra_options}
+ 
+    Ctn Run Command And Check Result As Strings    ${command}    ${expected_result}
+
+    Examples:        tc    extra_options                   expected_result    --
+            ...      1     --critical=15                   OK: Load average: 0.87, 0.63, 0.47 | 'load1'=0.87;;0:15;0; 'load5'=0.63;;;0; 'load15'=0.47;;;0; 
+            ...      2     --critical='2'                  OK: Load average: 0.87, 0.63, 0.47 | 'load1'=0.87;;0:2;0; 'load5'=0.63;;;0; 'load15'=0.47;;;0;
+            ...      3     --warning='1'                   OK: Load average: 0.87, 0.63, 0.47 | 'load1'=0.87;0:1;;0; 'load5'=0.63;;;0; 'load15'=0.47;;;0;
```

Comment:
Thresholds should be given by 3:
- for 1 minute average (load1)
- for 5 minute average (load5)
- for 15 minute average (load15)

And here the warning threshold should be chosen otherwise so that the WARNING status is reached.